INSTALL.md

```diff
@@ -26,6 +26,7 @@ targets.
  - Modern C++ compiler that supports `-std=c++14`
    - clang++ 3.5 or later _or_
    - g++-5 or later
+ - CMake (for build only)
  - Python 2.x (for build only)
  - [Node.js](https://nodejs.org/) (for build only)
  - [`pkg-config`](https://wiki.freedesktop.org/www/Software/pkg-config/) (for build only)
```

Comment:
It appears that we're no longer using `pkg-config` either? I was previously using this to ensure we were compiling and linking against a mason-installed version of Mesa, ran into trouble rebasing https://github.com/mapbox/mapbox-gl-native/commit/04fc3cb88389ee1ad607fceefd2fd2549addbb09 onto `master` because now it can't find `EGL/egl.h` (which it previously found through `pkg-config --cflags`)